internal/docs/exported_fields.go

```diff
@@ -74,15 +75,19 @@ func renderFieldsTable(builder *strings.Builder, collected []fieldsTableRecord)
 	builder.WriteString("\n")
 	for _, c := range collected {
 		description := strings.TrimSpace(strings.ReplaceAll(c.description, "\n", " "))
-		builder.WriteString(fmt.Sprintf("| %s | %s | %s |",
+		fmt.Fprintf(builder, "| %s | %s | %s",
 			escaper.Replace(c.name),
 			escaper.Replace(description),
-			c.aType))
+			c.aType)
+		if c.runtime {
+			builder.WriteString(" (runtime)")
+		}
+		builder.WriteString(" |")
 		if unitsPresent {
-			builder.WriteString(fmt.Sprintf(" %s |", c.unit))
+			fmt.Fprintf(builder, " %s |", c.unit)
 		}
 		if metricTypesPresent {
-			builder.WriteString(fmt.Sprintf(" %s |", c.metricType))
+			fmt.Fprintf(builder, " %s |", c.metricType)
```

Comment:
Update how this string is built by using fmt.Fprintf. Suggested by this static check QF1012 https://staticcheck.dev/docs/checks/#QF1012